src/stats.c

```diff
@@ -33,6 +34,42 @@ static struct {
     double user;
 } used_cpu;
 
+// log slow command
+#define BUILD_CMD_ITEM(cmd, type, access) #cmd,
+static const char * cmd_table[] = {CMD_DO(BUILD_CMD_ITEM)};
+static const size_t CMD_NUM = sizeof(cmd_table) / sizeof(char*);
+
+static uint32_t *counts_sum;
```

Comment:
`counts_sum` is only used in `stats_send_slow_log`, so how about declare `counts_sum` as a local variable in that function?